src/main/java/com/almanza/inventory/models/ShoeModel.java

```diff
@@ -10,11 +10,11 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
-public class PlayerDTO {
+public class ShoeModel {
     @JsonProperty("id")
     private Integer id;
     @JsonProperty("name")
     private String name;
-    @JsonProperty("number")
-    private Integer number;
+    @JsonProperty("brand")
+    private String brand;
```

Comment:
please include some property managing LocalDateTime, e.g: created
Also add an enumeration property, the enum class must have the Enum postfix